webapp/components/NotificationList/NotificationList.spec.js

```diff
@@ -3,8 +3,8 @@ import NotificationList from './NotificationList'
 import Notification from '../Notification/Notification'
 import Vuex from 'vuex'
 import Filters from '~/plugins/vue-filters'
-
 import Styleguide from '@human-connection/styleguide'
+import { notifications } from '~/components/utils/Notifications'
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-07T11:17:40Z" title="Thursday, November 7th 2019, 12:17:40 pm +01:00">Nov 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -3,8 +3,8 @@ import NotificationList from './NotificationList'
 import Notification from '../Notification/Notification'
 import Vuex from 'vuex'
 import Filters from '~/plugins/vue-filters'
-
 import Styleguide from '@human-connection/styleguide'
+import { notifications } from '~/components/utils/Notifications'
```


Hm, I see what you're doing here. Personally I would prefer to have a folder `NotificationList.spec/` where I would place the fake data. Now you're using this in many locations, also from other components.

I don't mind having it that way. Probably I would have put that in one components folder and simply imported it in another components spec.